profile_bluesky/startup/instrument/devices/aps_source.py

```diff
@@ -22,7 +22,6 @@
 class UndulatorEnergy(PVPositioner):
     """
     Undulator energy positioner.
-
```

Comment:
This vertical whitespace is a good practice.  When using Sphinx, it separates the summary line (used by [autosummary](https://www.sphinx-doc.org/en/master/usage/extensions/autosummary.html?highlight=autosummary)) from the remainder of the documentation content.  Is there a need to remove it from here (and other places in thie PR)?

Reply:
I'm not sure I understand this comment. Shouldn't I add a blank line to make it better for Sphinx? I think it doesn't like if the first like is too long.

Reply:
Right.  You *want* the blank line.  Github is showing that the blank line has been removed.